setup.py

```diff
@@ -22,7 +22,7 @@
     "lenses",
     "pyyaml",
     "jsonschema",
-    "dataclasses",  # for 3.6, as it ships with 3.7
+    'dataclasses;python_version=="3.6"',  # for 3.6, as it ships with 3.7
```

Comment:
Did you check somehow this is the exact correct syntax? So it doesn't break for "3.6.1"?

Reply:
I checked that it didn't break the setup procedure in general but it's a good question.

Reply:
Now it is correct